src/devices/HuskyLens/DataFrame.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.HuskyLens
+{
+    internal class DataFrame
+    {
+        private readonly byte[] _data;
+
+        public DataFrame(ReadOnlySpan<byte> data)
+        {
+            _data = data.ToArray();
+        }
+
+        public Command Command => (Command)_data[4];
+
+        public ReadOnlySpan<byte> Data => new ReadOnlySpan<byte>(_data, 5, _data[3]);
+
+        public bool IsValid()
+        {
+            return
```

Comment:
can you write a comment why are you expecting this specific data? Is this some kind of signature? How do I validate correctness?